### PR TITLE
Document LSP-volar

### PR DESCRIPTION
**Pull request by predragnikolic**
Wait till this [this PR](https://github.com/sublimelsp/repository/pull/53 ) is merged.

### CODE REVIEW

**Comment by predragnikolic** [comment]
Because I updated only the documentation, Github Actions will not be run. (so ignore the waiting status)

File: docs/src/language_servers.md [hunk 1]
```diff
@@ -559,8 +559,16 @@ For development using the Deno framework follow installation instructions on [LS
...
 
+There are multiple options:
+
+### Vetur
+
 Follow installation instructions on [LSP-vue](https://github.com/sublimelsp/LSP-vue).
 
+### Volar
+
+Follow installation instructions on [LSP-volar](https://github.com/sublimelsp/LSP-volar).
```
Comment (rchl) [inline docs/src/language_servers.md:570]: Maybe mention that Volar is only for Vue 3?

Comment (predragnikolic) [reply docs/src/language_servers.md:570]: Volar can be setup to work with Vue 2. (See the Volar readme for more details)

The only major difference that I have seen is that vetur supports monorepos, while volar supports the script setup syntax.

Comment (rchl) [reply docs/src/language_servers.md:570]: OK. A bit too much work to try. :)

**Review by rchl — APPROVED** [review]

**Pull request merged**